Update ElementRef example in RC/Blazor JS interop

guardrex opened the pull request:
Addresses #10717 
Fixes #11607

@pfeurean @dharmaturtle *Now, we're cook'in with gas!* :smile: What do you think of the updates here? In addition to fixing it, I align it better with the sample app. In a future common sample update, I'll (probably) roll this into the sample app. For now tho, let's just get this update merged asap.

dharmaturtle commented:
Addresses #11607 very nicely, thanks!

Just wondering, the `public async void SetFocus()` makes my neck hairs rise. Is this a safe spot to not return `Task`? I know in some places (like event handlers) that's accepted.

guardrex commented:
Yes ... AFAIK.

I'm sure engineering will issue a proper 💀 **_death threat_** :skull: to me to change it if needed. :smile: lol

Sorry for the delay on it. The conference kept us all fairly busy last week, including the engineering cats. Thanks again for opening the issue and letting me know about the problem.

guardrex commented:
@danroth27 When you get to this one, you might wish to look at the whole topic. There were several other Preview 3 updates on https://github.com/aspnet/Docs/pull/11314 that didn't get fully reviewed due to the conference.

danroth27 commented on aspnetcore/razor-components/javascript-interop.md:
```diff
@@ -210,57 +210,88 @@ As far as .NET code is concerned, an `ElementRef` is an opaque handle. The *only
     ElementRef username;
 
-    void SetFocus()
+    public async void SetFocus()
```
Return Task

danroth27 commented on aspnetcore/razor-components/javascript-interop.md:
```diff
@@ -210,57 +210,88 @@ As far as .NET code is concerned, an `ElementRef` is an opaque handle. The *only
         }
 
-        public static Task Focus(this ElementRef elementRef)
+        public Task Focus(ElementRef elementRef)
```
What's the reasoning behind making these all instance methods?

guardrex commented:
@danroth27 

> What's the reasoning behind making these all instance methods?

Probably just held over from the first version of the topic. I refactored the `Display` and `Prompt` methods out and dropped the `ElementRef` example that used an instance method.

It's ready for another :eyes: look.
